src/util/nidmap.c

```diff
@@ -1245,7 +1249,7 @@ int prrte_util_decode_ppn(prrte_job_t *jdata,
     }
 
     /* reset any flags */
-    for (m=0; m < jdata->map->nodes->size; m++) {
+    for (m=0; m < jdata->map->num_nodes; m++) {
```

Comment:
Same here again

Reply:
Is there a case where the jdata-> map will change though?

Reply:
Not sure I understand what that has to do with it - the map can't change while you are in this routine as it should only be called from within an event as it touches global data

Reply:
My thinking is this is a minor optimization since jdata->map shouldn't change.
but I guess it doesn't matter.

Reply:
Fixed.